src/components/dashboards/admin.js

```diff
@@ -0,0 +1,174 @@
+import React, { Component } from 'react';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Dropdown from 'react-bootstrap/Dropdown';
+import { connect } from 'react-redux';
+import Table from 'react-bootstrap/Table';
+import { Link } from 'react-router-dom';
+import { injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import Container from 'react-bootstrap/Container';
+import NavBar from '../sharedComponents/AuthNavBar';
+import '../../scss/usersTable.scss';
+import noUserimage from '../../assets/no-user-image.png';
+import Footer from '../sharedComponents/Footer';
+import { getUsersAction } from '../../redux/actions/viewUsers';
+import SideBar from '../sharedComponents/Sidebar';
+import { PopUp } from '../popUp';
+import { setRoleAction } from '../../redux/actions/setRole';
+import translate from '../languages/Translate';
+
+class AdminDashboard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      roles: ['manager', 'travel team member', 'requester', 'travel administrator', 'supplier', 'super administrator'],
+      popUp: 'invisible',
+      popUpActiveItem: {},
+    };
+    this.handleSetRolePopUp = this.handleSetRolePopUp.bind(this);
+    this.updateUsers = this.updateUsers.bind(this);
```

Comment:
You do not need to bind arrow functions.
Ideally, you will use bind if the function is declared as `handleSetRolePopUp()`

Reply:
I have corrected it